Database/Corrections/Classic/classicQuestFixes.lua

```diff
@@ -1621,7 +1623,38 @@ function QuestieQuestFixes:Load()
             [questKeys.questLevel] = 5, -- #2306
         },
         [5634] = {
+            [questKeys.startedBy] = {{376},nil,nil},
+            [questKeys.objectivesText] = {},
+            [questKeys.exclusiveTo] = {5635,5636,5637,5638,5639,5640},
+        },
+        [5635] = {
+            [questKeys.startedBy] = {{377},nil,nil},
+            [questKeys.exclusiveTo] = {5634,5636,5637,5638,5639,5640},
+        },
+        [5636] = {
+            [questKeys.exclusiveTo] = {5634,5635,5637,5638,5639,5640},
+        },
+        [5637] = {
+            [questKeys.startedBy] = {{1226},nil,nil},
+            [questKeys.exclusiveTo] = {5634,5635,5636,5638,5639,5640},
+        },
+        [5638] = {
+            [questKeys.exclusiveTo] = {5634,5635,5636,5637,5639,5640},
+        },
+        [5639] = {
+            [questKeys.exclusiveTo] = {5634,5635,5636,5637,5638,5640},
+        },
+        [5640] = {
+            [questKeys.name] = "Desperate Prayer",
             [questKeys.startedBy] = {{11401},nil,nil},
+            [questKeys.finishedBy] = {{376},nil},
+            [questKeys.requiredLevel] = 10,
+            [questKeys.questLevel] = 10,
+            [questKeys.requiredRaces] = bit.bor(raceIDs.HUMAN, raceIDs.DWARF),
```

Comment:
```suggestion
            [questKeys.requiredRaces] = raceIDs.HUMAN + raceIDs.DWARF,
```

This should be the correct format. The `or` connection is done in the code.

Reply:
Thanks! I was wondering how to handle this.